common/src/test/java/com/microsoft/identity/common/BrowserSelectorTest.java

```diff
@@ -253,14 +225,27 @@ public TestBrowserBuilder setVersion(String version) {
             return this;
         }
 
+        private PackageInfo addSignatures(final PackageInfo packageInfo, final Signature[] signatures) {
+            if (Build.VERSION.SDK_INT < Build.VERSION_CODES.P) {
+                packageInfo.signatures = signatures;
+                return packageInfo;
+            }
+
+            final SigningInfo signingInfo = mock(SigningInfo.class);
+            when(signingInfo.hasMultipleSigners()).thenReturn(false);
+            when(signingInfo.getSigningCertificateHistory()).thenReturn(signatures);
+            packageInfo.signingInfo = signingInfo;
+            return packageInfo;
+        }
+
         public TestBrowser build() {
             Signature[] signatures = new Signature[mSignatures.size()];
 
             for (int i = 0; i < mSignatures.size(); i++) {
                 signatures[i] = new Signature(mSignatures.get(i));
             }
 
-            final PackageInfo pi = Util.addSignatures(new PackageInfo(), signatures);
```

Comment:
Is the Util method still there? Can we delete it if there are no usages?

Reply:
It's still used.